execution/executor-benchmark/src/pipeline.rs

```diff
@@ -152,4 +197,33 @@ where
             handle.join().unwrap()
         }
     }
+
+    pub fn partition_block(
```

Comment:
Seems like this function is not being called?

Reply:
fixed